apps/trustlab/src/components/Error/Error.js

```diff
@@ -0,0 +1,75 @@
+import { Section } from "@commons-ui/core";
+import { Figure, Link } from "@commons-ui/next";
+import { LexicalRichText } from "@commons-ui/payload";
+import { Typography, Box, Button } from "@mui/material";
+import React, { forwardRef } from "react";
+
+// eslint-disable-next-line import/no-unresolved
+import ErrorPageIcon from "@/trustlab/assets/error-page-icon.svg?url";
+
+const ErrorPage = forwardRef(function ErrorPage(props, ref) {
+  const { title, subtitle, image, link, ...other } = props;
+  return (
+    <Section
+      ref={ref}
+      {...other}
+      sx={{
+        py: 8,
+        textAlign: "center",
+        margin: "0 auto",
+        px: { xs: 2.5, md: 0 },
+      }}
+    >
+      <Box
+        display="flex"
+        justifyContent="center"
+        alignItems="center"
+        flexDirection="column"
+        sx={{ maxWidth: 600, margin: "0 auto" }}
+        gap={2.5}
+      >
+        <Figure
+          ImageProps={{
+            alt: "Error page background",
```

Comment:
If it's a _background_ image, do we need to use Figure? Why not just set it via [css](https://developer.mozilla.org/en-US/docs/Web/CSS/background-image)